src/app/components/TranscriptExperiment/SignPost/index.tsx

```diff
@@ -0,0 +1,63 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
+import { useContext, useId } from 'react';
+import { ServiceContext } from '#app/contexts/ServiceContext';
+import Text from '#app/components/Text';
+import VisuallyHiddenText from '#app/components/VisuallyHiddenText';
+import styles from './index.styles';
+import SVGs from './svgs';
+
+const DEFAULT_MESSAGE =
+  'Help reduce your power and data usage by not playing video content.';
+const DEFAULT_LOAD_TITLE = 'Load Video';
+
+type Props = {
+  title?: string;
+};
+
+const SignPost = ({ title = '' }: Props) => {
```

Comment:
Could we rename this component - is it more like a Transcript CTA perhaps? A bit like our LiteSiteCTA? Or perhaps something more descriptive around DataSaving / DataUsage?